server/tests/lib/util_test.py

```diff
@@ -1776,12 +1776,13 @@ def test_fetch_highest_coverage_with_no_observation_dates(
 
 
 class TestFeatureFlagsTest(unittest.TestCase):
-  FEATURE_FLAG_COUNT = 6
```

Comment:
@gmechali Just a FYI for this modification. Let me know if this might have unintended consequences.